ports/ffmpeg/portfile.cmake

```
@@ -613,6 +633,20 @@ set(ENV_LIB_PATH "$ENV{${LIB_PATH_VAR}}")
 
 message(STATUS "Building Options: ${OPTIONS}")
 
+# on apple platforms we might need to do multiple builds, and then
+# join them using lipo - ffmpeg does _not_ support multi-arch builds
+set(BUILD_SCRIPT "${CMAKE_CURRENT_LIST_DIR}/build.sh.in")
+if ("${VCPKG_CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
```

```suggestion
if (VCPKG_TARGET_IS_OSX)
```

(for consistency with how this is checked in other places)

---

Makes sense, changed.